test/e2e/plain-v0_provisioner_test.go

```diff
@@ -86,22 +90,59 @@ var _ = Describe("plain-v0 provisioner", func() {
 			})
 		})
 
-		// TODO(tflannag): add test specs for ensuring the object and metadata ConfigMaps get re-created when deleted
+		It("should re-create the bundle configmaps", func() {
+			var (
+				cm *corev1.ConfigMap
+			)
+
+			By("getting the metadata configmap for the bundle")
+			configMapTypeRequirement, err := labels.NewRequirement("core.rukpak.io/configmap-type", selection.Equals, []string{"metadata"})
```

Comment:
For follow-up/TODO: add a constant for these labels?

Reply:
I just ended up making a TODO comment and pushing that up. I think this is a general issue across all packages, so hopefully we can have establish some consistency throughout this repository vs. continuing to hardcode labels over time.

Reply:
Added an action that creates issues from TODO comments in #106.